src/lib/ext-activitylog.js

```diff
@@ -251,6 +266,16 @@ class Controller {
       if (existingLogs.length) {
         this.handleNewLogs(existingLogs);
       }
+
+      if (filterTabId) {
+        this.view.logHeading.textContent = `${this.view.logHeading.textContent} Filtered By Tab Id: ${filterTabId}`;
```

Comment:
As it is this looks a bit of a violation of the MVC pattern, the controller shouldn't change the textContent of the an element part by the view by directly accessing it as a property.

isn't the view already receiving the filterTabId as part of the filter properties got from the model?
if that is the case, then the view could use that to update also that header element accordingly.

Reply:
> isn't the view already receiving the filterTabId as part of the filter properties got from the model?
> if that is the case, then the view could use that to update also that header element accordingly.

The `log-view` web component is directly getting the filter object from model via `isFilterMatched` method. 
I am made a new method in new to deal with it. Does that work?